File: Day1_Java/q20.java
import java.util.Scanner;

public class q20 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int amount = sc.nextInt();
        int dis = 0;
        // if (amount >= 0 && amount <= 5000)
        // System.out.println("Total Payable Amount" + (amount - (0 * amount) / 100));
        // else if (amount > 5000 && amount <= 7000) {
        // System.out.println(amount - (5 * amount) / 100);
        // }

        if (amount >= 0 && amount <= 5000)
            dis = 0;
        else if (amount > 5000 && amount < 7000)
            dis = 5;
        else if (amount > 7000 && amount < 9000)
            dis = 10;
        else
            dis = 20;
        System.out.println(amount - (dis * amount) / 100);
    }
}

File: Questions_in_jS_Console_Based/q20.js
//Shop discount


let money = prompt('Enter the price To get the Discount..');

let dis = 0;
(() => {

    if (money >= 0 && money <= 5000) dis = 0;
    else if (money > 5000 && money <= 7000) dis = 5;
    else if (money > 7000 && money <= 10000) dis = 10;
    else if (money > 10000) dis = 25;
    else
        alert("invalid Input Please Check your Price.")

    let discountprice = money - (dis * money) / 100;
    
    alert(`Discount: ${discountprice} & Total Payable Amount is: ${money}`);
})();

